src/pages/likedvideoPage/likedvideo.js

```diff
@@ -1,14 +1,22 @@
-import { Navbar, Sidebar, Card } from "../../components";
+import { Card, EmptyComp, Navbar, Sidebar } from "../../components";
+import { useGlobal } from "../../context/globalContext";
 
 export const LikedVideo = () => {
+  const {
+    globalState: { likeVideo },
+  } = useGlobal();
+
   return (
     <div>
       <Navbar />
       <section className="main-box">
         <Sidebar />
         <main className="main-product">
+          {likeVideo.length === 0 && <EmptyComp />}
```

Comment:
You can name it 'EmptyLikesPage' or 'PagePlaceholder'  in general.

Reply:
thanks for the feedback i fixed it